datasets/1.0/huggingface-rag-dataset/metadata.json

```diff
@@ -40,9 +42,43 @@
     "separator": "cr:separator",
     "source": "cr:source",
     "subField": "cr:subField",
-    "transform": "cr:transform"
+    "transform": "cr:transform",
+    "arrayShape": "cr:arrayShape",
+    "isArray": "cr:isArray",
```

Comment:
For readability, can we follow the alphabetical order in all `@context`?

(Can be done in another PR)

Reply:
Done.

Reply:
Done.